engines/docker/resultset.go

```diff
@@ -92,13 +92,6 @@ func (r *resultSet) ExtractFile(path string) (ioext.ReadSeekCloser, error) {
 }
 
 func (r *resultSet) ExtractFolder(path string, handler engines.FileHandler) error {
-	// We'll treat paths ending with a slash as paths to folders
-	if !strings.HasSuffix(path, "/") {
-		debug("ExtractFolder(%s) doesn't end with '/' which can't be a folder", path)
-		return runtime.NewMalformedPayloadError(fmt.Sprintf(
-			"docker folder path: '%s' doesn't end with slash, paths to folders must end with a slash", path,
-		))
-	}
 	return r.extractResource(path, func(name string, stream ioext.ReadSeekCloser) error {
```

Comment:
Please add `path+"/"` and fix engine test cases to not use paths ending in slash :)